.gitignore

```diff
@@ -3,7 +3,6 @@ __pycache__/
 *.py[cod]
 *$py.class
 .vscode/
-.env
 
```

Comment:
Why did you decide to have `.env` removed from the gitignore file

Reply:
Hey, by default the file had already been ignored hence that was just a repetition.